src/locales/default.json

```diff
@@ -5,6 +5,8 @@
   "backToHome": "Home",
   "actions": "Actions",
   "results": "Results",
+  "resultsError": "Results could not be processed. This is most likely due to a malconfigured strategy or a performance issue with an involved contract. Please make sure none of the above applies before contacting support.",
```

Comment:
```suggestion
  "resultsError": "Results could not be processed. This is most likely due to a misconfigured strategy or a performance issue with an involved contract.",
```

Reply:
Yeeeeees I know that was all a bit long. :D The idea is to reduce helpdesk requests for issues we are not responsible for, thus the "Please make sure..." part but well,... might work just good enough without it. At least it doesn't look completely broke now anymore.